### PR TITLE
add feedback get and post

### PR DESCRIPTION
**Pull request by petitminion**
close https://github.com/metabrainz/pylistenbrainz/issues/21

### CODE REVIEW

**Comment by petitminion** [comment]
hello @mayhem  :) could we merge this ?

**Comment by petitminion** [comment]
hello @mayhem  :) could we merge this ?

File: pylistenbrainz/client.py [hunk 1]
```diff
@@ -219,6 +219,27 @@ def submit_playing_now(self, listen):
         """
         return self._post_submit_listens([listen], LISTEN_TYPE_PLAYING_NOW)
 
+    def submit_feedback(self, feedback, recording_mbid):
```
Comment (mayhem) [inline pylistenbrainz/client.py:222]: I think this function should be submit_user_feedback() -- that is the only thing that I see that we ought to change. Thanks!

**Review by mayhem — CHANGES_REQUESTED** [review]
One tiny change and we're good to go.

File: pylistenbrainz/client.py (hunk 1 above)
Comment (petitminion) [reply pylistenbrainz/client.py]: @mayhem done :)

Comment (petitminion) [reply pylistenbrainz/client.py]: @mayhem done :)

Comment (mayhem) [reply pylistenbrainz/client.py]: Thanks!

**Review by mayhem — APPROVED** [review]

File: pylistenbrainz/client.py (hunk 1 above)
Comment (petitminion) [reply pylistenbrainz/client.py]: @mayhem thx for merging :) is there a plan to make a v0.5.3 release tag  ?

**Comment by mayhem** [comment]
Done.

**Pull request merged**